pytest_container/helpers.py

```diff
@@ -157,3 +159,26 @@ def get_always_pull_option() -> bool:
 
     """
     return bool(int(os.getenv("PULL_ALWAYS", "1")))
+
+
+def run_command(
+    cmd: list[str],
```

Comment:
This doesn't work in python < 3.9 (plus import it at the top if not present)
```suggestion
    cmd: List[str],
```